Game-as-a-Service/HUNTER-Gungi
# Feat/mongodb server

# Description
Added `mongodb` to `docker-compose.local.xml` and connected mongodb instance to nestjs db connection provider.

# How to test
1. Clean all local Docker artifacts with `npm run docker:clean` on the root folder level
2. Run `npm start` on the root folder level to start the application
3. You should be able to see that the server starts on `localhost:8000`

# New Dependencies
N/A

<img width="1272" alt="image" src="https://user-images.githubusercontent.com/4503670/236990536-1c43a989-1524-4c2f-a6cc-99a1ae26cf90.png">
